### PR TITLE
Feature/positional param ordering

### LINKED ISSUE #51: Improve usability of cmdlets by eliminating redundant parameters
In the world of docker, the image name comprises of the full text that includes these elements:
* registry
* image name
* tag / digest
as `<registry>/<image name>/<tag / digest>`

But the parameters the cmdlets ask for, are 3 separate parameters as stated above.

To improve usability, I think we should let the users formulate the complete image names themselves, than to have this logic added in the cmdlets.

This will eliminate the need for 3 parameters, and streamline them into 1, similar to how docker cli used to accept.

In short, eliminate these parameters:
* `-Registry`
* `-ImageName`
* `-Tag`

### LINKED ISSUE #78: Positional args for all relevant cmdlets
so it's easier to just do the right thing.

### PR DESCRIPTION
**Pull request by rasmusjelsgaard**
* Quality of life improvement. Lets you omit the param name of the first mandatory parameter, which is often all you need to specify.

Closes #78

### CODE REVIEW

**Comment by zionyx** [comment]
Until `-Registry` and `-Tag` are eliminated. 🤩

**Comment by rasmusjelsgaard** [comment]
> This looks like a good improvement. But this does not fix #51 however. 😅

You are right. I linked to the wrong issue. It closes #78

File: Source/Public/Format-DockerTag.ps1 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 function Format-DockerTag {
-    [CmdletBinding()]
+    [CmdletBinding(PositionalBinding = $false)]
     param (
-        [Parameter(ValueFromPipelineByPropertyName = $true)]
+        [Parameter(ValueFromPipelineByPropertyName = $true, Position = 0)]
```
Comment (zionyx) [inline Source/Public/Format-DockerTag.ps1]: Very nice! :sun_with_face:

File: Source/Public/Invoke-DockerPull.ps1 [hunk 2]
```diff
@@ -1,16 +1,16 @@
 . "$PSScriptRoot\..\Private\Write-PassThruOutput.ps1"
 
 function Invoke-DockerPull {
-    [CmdletBinding()]
+    [CmdletBinding(PositionalBinding = $false)]
     param (
         [Parameter(ValueFromPipelineByPropertyName = $true)]
         [String]
         $Registry = '',
```
Comment (zionyx) [inline Source/Public/Invoke-DockerPull.ps1]: `Invoke-DockerPush.ps1` and `Invoke-DockerTag.ps1` have `$Registry` reordered as the 2nd parameter. 

Consider applying the same coherency for this cmdlet too?

Comment (rasmusjelsgaard) [reply Source/Public/Invoke-DockerPull.ps1]: My idea was to only do it for params where a convention already exists as with Dockerfile and build context.

**Pull request merged**